src/data/hierarchy_data_query.sql

```diff
@@ -1,5 +1,4 @@
-
-  USE ROLE ACCOUNTADMIN;
+USE ROLE ACCOUNTADMIN;
```

Comment:
It's important to have the first blank line and to indent the USE ROLE ACCOUNTADMIN; or the modal displays incorrectly.

Reply:
nevermind, I actually think this change looks better. Nice job.